### ISSUE TITLE
Update bold color contrast

### ISSUE THREAD

**Issue by kyrolyte** [lyteword/lyteword#32]
- Bold elements currently render as a soft blue color similar to the color of links
- The contrast ratio against the background is insufficient and needs to be adjusted for readability 
- In addition, the logo of the website when switched to dark-mode has an insufficient contrast and will need to be adjusted
